spec/models/comment_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:test_movie) { Movie.new }
+  let(:test_user) { User.create!(name: "Carlos", email: "Carlos@gmail.com", password: "password" ) }
+
+  describe 'validations' do
+
+    before(:each) do
+      test_movie.get_movies_by_title("Fight Club")
+      first_movie = Movie.first
+      @new_comment = first_movie.comments.new(content: "This film was a work of art", user_id: test_user.id)
+    end
+
+    it 'saves a new comment if successfully created' do
+      expect{@new_comment.save}.to change{Comment.count}.by(1)
+    end
+
+    it 'saves a new comment if content is present' do
+      if @new_comment.content = !nil
```

Comment:
When testing, since you are in complete control of the context of each test, you should not need any conditional logic in any of your tests. Instead, have different tests for different contexts. E.g.: have one test where @new_comment.content exists, and another test where @new_comment.content does not exist. Given-->Then-->When